msrest/paging.py

```diff
@@ -123,7 +123,8 @@ def advance_page(self):
         :return: The current page list
         :rtype: list
         """
-        if self.next_link is None:
+        # Will test None or "". ARM says None only, but there is instances with ""
+        if not self.next_link:
```

Comment:
@lmazuel, i believe @iscai-msft tried this last week, and it ended up getting nothing at all. The patch i mentioned in the #147 works. With that I feel the paging code logic looks a bit odd worthwhile to revisit.
Either way if Advisor team decides to take service end fix, then this PR is not important any more.

Reply:
Yes, it doesn't pass CI it's too naive indeed. I didn't realize #147 exists, but I think it's not the right fix. Let me think more on this.